.github/workflows/build.yml

```diff
@@ -60,10 +50,51 @@ jobs:
         with:
           name: age-binaries
           path: age-*
+  sign:
+    name: Sign Windows binaries
+    if: ${{ github.event_name == 'release' }}
```

Comment:
This conditional could still be the "if repository is this canonical one", that way the CI code that does the signing gets a workout and is known to work before a release cycle hits. Also it would make it easier for some people to test intermediary builds.

Reply:
If it's a PR it still will run in canonical but without the secret, so I had to get a little specific with the logic in the newest revision.